app-benchmarks/iozone/iozone-3.506-r1.ebuild

```diff
@@ -54,6 +54,10 @@ src_configure() {
 	# Otherwise it uses K&R function declaration where ints are sometimes omited
 	# https://bugs.gentoo.org/894334
 	append-cppflags -DHAVE_ANSIC_C
+
+	case ${ARCH} in
+		x86) append-cflags -D_FILE_OFFSET_BITS=64 ;; # bug #948362
```

Comment:
Why doesn't append-lfs-flags work? We should figure that out instead. x86 isn't the only arch that will be affected.

Maybe we need to append CPPFLAGS into CFLAGS in this ebuild.